src/js/components/covid19/amountsVisualization/amounts/remainingBalance/LinesAndText.jsx

```diff
@@ -0,0 +1,232 @@
+/**
+ * LinesAndText.jsx
+ * created by Jonathan Hill 04/26/21
+ */
+
+import React, { useState, useEffect, useLayoutEffect, useRef } from 'react';
+import { upperFirst } from 'lodash';
+import PropTypes from 'prop-types';
+
+import {
+    calculateUnits,
+    formatMoneyWithPrecision
+} from 'helpers/moneyFormatter';
+
+import {
+    amountsPadding,
+    rectangleMapping,
+    startOfChartY,
+    rectangleHeight,
+    remaniningBalanceLineWidth,
+    lineStrokeWidth,
+    heightOfRemainingBalanceLines,
+    remainingBalanceLineWidth,
+    defaultLineData
+} from 'dataMapping/covid19/amountsVisualization';
+
+import { defaultTextState } from 'helpers/covid19/amountsVisualization';
+
+import TextGroup from '../shared/TextGroup';
+
+const propTypes = {
+    scale: PropTypes.func,
+    overviewData: PropTypes.object,
+    displayTooltip: PropTypes.func,
+    hideTooltip: PropTypes.func,
+    lineData: PropTypes.object,
+    rectangleData: PropTypes.object,
+    dataId: PropTypes.string,
+    width: PropTypes.number,
+    setRemainingBalanceLabelData: PropTypes.func,
+    setRemainingBalanceValueData: PropTypes.func
+};
+
+const LinesAndText = ({
```

Comment:
rename to BracketsAndText? Might help to rename this one for readability to distinguish it as distinct from the bars which have the vertical lines!

Reply:
Not a bad idea. There are no brackets though? There are just lines and text. And a user can determine what the lines are by viewing the chart and seeing where they are in the file structure since this is in the `remainingBalance` folder.